### DISCUSSION TITLE
Why can't I pass the student certification

### DISCUSSION THREAD

**Discussion by Luochancy** [community/community#50242] [GitHub Education] [answered]
I am a vocational high school student from China. I have submitted my student certificate from the school within a few days. But I keep prompting You are significantly more likely to be verified if you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio. I also tried to modify my bio and name to match this error, but it just didn't work. Even if I submit my ID card together.

**Comment by samus-aran** [comment, marked as answer]
Hi @Hacker-yfs 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process. Do you use a variation of your name or a nickname? Is in another language?  It’s these small but important points to consider when applying that can make a much smoother process.

Hope this helps. Let us know how you get on 👍

**Reply by Luochancy** [reply]
May I ask if there is any problem in the materials I submitted?

**Reply by Luochancy** [reply]
I have tried to change my user name to my real name, but this error still appears.

**Reply by aviiciii** [reply]
Same problem with me!

**Reply by SuperdoseLuminal** [reply]
I also can't pass. I am student from Singapore. Although I submit my student ID or my offer letter, the certification still can't pass.

**Reply by sascharo** [reply]
> I also can't pass. I am student from Singapore. Although I submit my student ID or my offer letter, the certification still can't pass.

Did it work in the end? I'm from NTU and have had no issues in the previous 2+ years to apply successfully. But in the past few months, something has changed; this time, I couldn't automatically pass with my NTU email address.

**Reply by SuperdoseLuminal** [reply]
I pass, I use the admission certificate from Gus's website.
Offer Can't pass.

**Reply by SuperdoseLuminal** [reply]
I misremembered, I submitted a Student Status Letter.

**Reply by sascharo** [reply]
Me too; I submitted a NTU letter of certification (student status) a few days ago. Though, so far, I haven't heard anything back. In previous years, it was an instant approval with an @e.ntu.edu.sg email address.
